src/Core/src/ImageSources/Android/Glide/RequestBuilderExtensions.cs

```diff
@@ -63,5 +63,29 @@ public bool OnResourceReady(Java.Lang.Object result, Java.Lang.Object model, ITa
 				return false; // True would prevent target.OnResourceReady from being called - not necessary here
 			}
 		}
+
+		internal class RequestCompleteListener : Java.Lang.Object, Bumptech.Glide.Request.IRequestListener
```

Comment:
Wanted a listener that didn't send back the Drawable... maybe this isn't really necessary as we can just discard the drawable later anyway? Seemed better to not care about the drawable sooner in the flow though.